diablo/jobs/admin_emails_job.py

```diff
@@ -61,39 +56,15 @@ def key(cls):
         return 'admin_emails'
 
     def _alert_admin_of_instructor_changes(self):
-        approval_uids_per_section_id = self._get_approval_uids_per_section_id()
         for course in self.courses:
-            instructor_uids = [instructor['uid'] for instructor in course['instructors']]
-            approval_uids = approval_uids_per_section_id[int(course['sectionId'])]
-            all_instructors_have_approved = all(uid in approval_uids for uid in instructor_uids)
-            if not all_instructors_have_approved:
+            if course['scheduled'] and course['scheduled']['hasObsoleteInstructors']:
                 self._notify(course=course, template_type='admin_alert_instructor_change')
 
     def _alert_admin_of_room_changes(self):
-        scheduled_rooms_per_section_id = self._get_scheduled_locations_per_section_id()
-        locations_per_section_id = dict((c['sectionId'], c['meetingLocation']) for c in self.courses)
         for course in self.courses:
-            section_id = course['sectionId']
-            location = locations_per_section_id[section_id]
-            if location != scheduled_rooms_per_section_id[int(section_id)]:
+            if course['scheduled'] and course['scheduled']['hasObsoleteRoom']:
                 self._notify(course=course, template_type='admin_alert_room_change')
 
-    def _get_approval_uids_per_section_id(self):
-        section_ids = [s.section_id for s in self.all_scheduled]
-        all_approvals = Approval.get_approvals_per_section_ids(section_ids=section_ids, term_id=self.term_id)
-        approval_uids_per_section_id = {section_id: [] for section_id in section_ids}
-        for approval in all_approvals:
-            approval_uids_per_section_id[approval.section_id].append(approval.approved_by_uid)
-        return approval_uids_per_section_id
-
-    def _get_scheduled_locations_per_section_id(self):
-        locations_per_section_id = {}
-        rooms = Room.get_rooms([s.room_id for s in self.all_scheduled])
-        locations_per_room_id = dict((room.id, room.location) for room in rooms)
-        for section_id, room_id in dict((s.section_id, s.room_id) for s in self.all_scheduled).items():
-            locations_per_section_id[section_id] = locations_per_room_id[room_id]
-        return locations_per_section_id
-
```

Comment:
We're losing a lot of code here in order to align with the logic already being used by the course changes page.